__tests__/app.test.js

```diff
@@ -25,3 +25,22 @@ describe("GET /api", () => {
       });
   });
 });
+
+
+describe("GET /api/topics", () => {
+  test("gives status of 200 and responds with an object of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { allTopics } }) => {
+        allTopics.forEach((topic) => {
+					expect(Object.keys(topic)).toHaveLength(2);
```

Comment:
This object.keys could make this test brittle, you've correctly used matchObject to check that the keys exist and leave room for new ones in the future. If you did add more keys, then the expect here would break.